src/handlers.js

```diff
@@ -10,6 +10,33 @@ const {consume} = require('taskcluster-lib-pulse');
 const debugPrefix = 'taskcluster-github:handlers';
 const debug = Debug(debugPrefix);
 
+const TITLES = { // maps github checkruns statuses and conclusions to titles to be displayed
+  success: 'Success',
+  failure: 'Failure',
+  neutral: 'It is neither good nor bad',
+  cancelled: 'Cancelled',
+  timed_out: 'Timed out',
+  action_required: 'Action required',
+  queued: 'Queued',
+  in_progress: 'In progress',
+  completed: 'Completed',
+};
+
+const CONCLUSIONS = { // maps status communicated bu the queue service to github checkrun conclusions
+  completed: 'success', // TODO: remove this stupid rule from es-lint
+  failed: 'failure',
+  exception: 'failure',
+  'deadline-exceeded': 'timed_out',
+  canceled: 'cancelled',
+  superseded: 'neutral', // is not relevant anymore
+  'claim-expired': 'failure',
+  'worker-shutdown': 'neutral', // will be retried
```

Comment:
I think this and `intermittent-task` are also failures if they happen on the last run attempt. @djmitche might be able to correct me though.

Reply:
I'm comfortable continuing to call it neutral even in that case -- it wasn't the code's fault.